src/classes/PS_Request.cls

```diff
@@ -64,9 +64,31 @@ public inherited sharing class PS_Request {
     public static String JWT_SFDO_ID_PARAM = 'sfdo.id';
     public static String JWT_PRODUCT_PARAM = 'sfdo.product';
     public static String JWT_ROLES_PARAM = 'sfdo.roles';
+    
+    public static String ElevateEndpointTokenize = '/v1/payments/verified/tokens';
+    public static String ElevateEndpointPurchase = '/v1/payments/verified/purchase';
+    public static String ElevateEndpointCommitment = '/v1/payments/verified/commitments';
+    public static String ElevateEndpointCommitmentCancel = '/v1/payments/verified/commitments/{0}/cancel';
 
     private static Integer RECOMMENDED_TIMEOUT_MS = 20000;
 
+    /***
+    * @description Returns payments service endpoint URL
+    * @param endpoint Elevate Endpoint enum 
+    * @return String
+    */
+    public String getEndpoint(ElevateEndpoint endpoint) {
```

Comment:
@kenlewis-sfdo , feel free to create a new tech debt/refactoring WI for the common class renaming, for example `PS_Request` -> `PS_PaymentRequest` . Renaming can be done on any purchase (`GE_PaymentsServices`) or commitment request  (`PS_CommitmentRequest`) related work. Or just feel free to rename/refactor classes (if at all) when working on a PR that changes `GE_PaymentsServices` class. CC @psadie-sfdo , @ptewson-sfdo